### PR TITLE
fix: add shivamp1998

### LINKED ISSUE #1454: Sarvesh Pandey
## Profile Screenshot

image

### PR DESCRIPTION
**Pull request by shivamp1998**
resolves #1454

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/shivamp1998.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image" : "https://user-images.githubusercontent.com/48366063/193478291-efc11969-6ba4-41bb-bfdf-89d1644afdba.png",
+    "issueId" : 1454,
+    "name" : "sarvesh pandey",
+    "username" : "shivamp1998"
+}
```
Comment (Panquesito7) [inline profiles/shivamp1998.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/shivamp1998.json (hunk 1 above)
Comment (shivamp1998) [reply profiles/shivamp1998.json]: added newline

File: profiles/shivamp1998.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image" : "https://user-images.githubusercontent.com/48366063/193478291-efc11969-6ba4-41bb-bfdf-89d1644afdba.png",
+    "issueId" : 1454,
+    "name" : "sarvesh pandey",
+    "username" : "shivamp1998"
```
Comment (Panquesito7) [inline profiles/shivamp1998.json:5]: It's best to remove the spaces before `:`.
```suggestion
    "image": "https://user-images.githubusercontent.com/48366063/193478291-efc11969-6ba4-41bb-bfdf-89d1644afdba.png",
    "issueId": 1454,
    "name": "sarvesh pandey",
    "username": "shivamp1998"
```

Comment (shivamp1998) [reply profiles/shivamp1998.json]: Done, please review!

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Comment by shivamp1998** [comment]
Is it possible to add hacktoberfest label to this?

**Pull request merged**